tutorial/conan_repositories/setting_up_conan_remotes/artifactory/artifactory_ce_cpp.rst

```diff
@@ -17,20 +17,56 @@ Running Artifactory CE
 
 There are several ways to run Artifactory CE:
 
-* **Running from a Docker image:**
+Running from a Docker image
```

Comment:
This is great, but I think what we really want to have is:

- changing our ``downloads`` page, with an updated way to launch Artifactory
- Maybe create our own ``docker compose`` or similar that wraps those details (at least for a quick getting started)?